config/jobs/kubeflow/kubeflow-postsubmits.yaml

```diff
@@ -210,6 +237,8 @@ postsubmits:
     annotations:
       testgrid-dashboards: sig-big-data
       description: Postsubmit tests for kubeflow/arena.
+      testgrid-alert-email: cloud-ml-pipeline-oncall@google.com
+      testgrid-num-failures-to-alert: "3"  
   kubeflow/pipelines:
   - name: kubeflow-pipeline-postsubmit-standalone-component-test
```

Comment:
Hi @hilcj ! only tests that include "kubeflow-pipeline-" in their names are related to Kubeflow Pipelines. You can remove all others, because they are Kubeflow general tests.

Reply:
Got it.  Thanks.